playbook/app/pb_kits/playbook/tokens/_colors.scss

```diff
@@ -114,7 +114,7 @@ $shadow_colors: (
 
 /* Text colors ------------------------*/
 $text_lt_default:     $charcoal;
-$text_lt_light:       #919EAB;
+$text_lt_light:       #687887;
```

Comment:
![image](https://user-images.githubusercontent.com/863031/113170829-8b581300-920c-11eb-981e-7aaac751cf81.png)
I'm thinking let's hold off on the body light. Everything is starting to look the same.